### ISSUE TITLE
WCAG - documents missing language tag

### ISSUE THREAD

**Issue by glittle** [HTBox/allReady#529]
Need to add language attribute.

> Document language missing
> The language of the document is not identified.
> Why It Matters
> Identifying the language of the page allows screen readers to read the content in the appropriate language. It also facilitates automatic translation of content.
> How to Fix It
> Identify the document language using the `<html lang>` attribute (e.g., `<html lang="en">`).
> The Algorithm... in English
> The <html lang> attribute is missing or is empty.
> Standards and Guidelines
> 3.1.1 Language of Page (Level A)

**Comment by glittle** [comment]
Working on this...
